src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala

```diff
@@ -15,7 +15,7 @@ import org.broadinstitute.dsde.agora.server.model.AgoraEntity
 import spray.json._
 
 object AgoraMongoDao {
-  val MongoDbIdField = "_id"
+  val MongoDbIdField = "_snapshotId"
```

Comment:
Do we want to keep this one as _id? We really just want the field name in the methods entity to be called snapshotId. Here we're changing a field name in the counter collection.